spec/integ/bridge.spec.js

```diff
@@ -705,6 +705,7 @@ function mkMockMatrixClient(uid) {
         if (method === "POST" && path === "/register") {
             return client.register(data.user);
         }
+        return undefined;
```

Comment:
I'm surprised this is necessary

Reply:
The linter made me do it^^ …But I would have done it the same way anyway as it is more explicit in what is happening. Null(/undefined) values are a pita even if they are in plain sight.

Reply:
Fair :)